serving/docker/tensorrt-llm.Dockerfile

```diff
@@ -0,0 +1,95 @@
+# -*- mode: dockerfile -*-
```

Comment:
Why not just update FT Dockerfile?

Reply:
Ft has a lot of dependencies need to rebuild from source, not easy to upgrade